frontend/src/components/common/Label.tsx

```diff
@@ -171,7 +175,7 @@ export function HoverInfoLabel(props: HoverInfoLabelProps) {
 
   return (
     <LightTooltip title={hoverInfo || ''}>
-      <Typography className={classes.noWrap}>
+      <Typography className={`${classes.noWrap} ${classes.display}`}>
```

Comment:
In cases like this, please use `clsx(classes.noWrap, classes.display)` for combining classes.

Reply:
Opted for this solution